db/migrate/20220721134828_add_role_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string
+    add_column :users, :default, :string
+    add_column :users, :user, :string
+  end
+end
```

Comment:
- [ ] I got an error while running your migrations. It happened that you have two migration files with the same name. Kindly remove the one you're not using. I simply deleted this one so that I can successfully migrate your database without errors. So please fix this. 

- [ ] Your Authorization worked perfectly kudos. Just fix the migration issue for your project to be approved, okay 😄
 
<img width="279" alt="Code_KQqu5GPpur" src="https://user-images.githubusercontent.com/81468675/180488726-b9167114-3ce7-4874-a5ec-d707069c9963.png">

Reply:
Hello @julie-ify, thanks for your review. I've just fixed the issue.